.github/workflows/end2end.yaml

```diff
@@ -318,10 +319,37 @@ jobs:
           cache-to: type=gha,mode=max,scope=$GITHUB_REF_NAME-test
   
   build-ctst-test-image:
+
+    
+
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GIT_ACCESS_TOKEN }}
+          # repository: scality/Zenko
+      - name: Configure GIT
+        run: |
+          git config --global url.https://x-access-token:${{ secrets.GIT_ACCESS_TOKEN }}@github.com/.insteadOf https://github.com/
+          git config --global url.https://x-access-token:${{ secrets.GIT_ACCESS_TOKEN }}@github.com/.insteadOf git@github.com:
+          git config --global url.https://x-access-token:${{ secrets.GIT_ACCESS_TOKEN }}@github.com/.insteadOf ssh://git@github.com/
+        env:
+          GIT_ACCESS_TOKEN: ${{ secrets.GIT_ACCESS_TOKEN }}
+        
+      # - name: Test git token on cli-testing repo
+      #   run: git clone git@github.com:scality/cli-testing.git
+      # # - name: Test git token on public repo
+      # #   run: git clone git@github.com:scality/Zenko.git
+      # - name: Test git token on private repo
+      #   run: git clone git@github.com:scality/vault2.git
+      
+      - name: Install subfolder dependencies
+        working-directory: tests/ctst
+        run: yarn install
+      - name: Lint subfolder
+        working-directory: tests/ctst
+        run: yarn lint
```

Comment:
The workflow step says `build-ctst-test-image`. We are doing much more than that in this step, hence the name can be mis-leading and someone facing the issue will have to check the this workflow to figure out whats going wrong.
Either we should rename the step of use another one for this static test.

Similarly, we should change the name from `name: Lint subfolder` to something more logical as reading this I do not know which subfolder and will have to check the workflow.
May be `name: Lint ctst tests`